twilio/rest/accounts/v1/docs/DefaultApi.md

```diff
@@ -53,8 +52,8 @@ Name | Type | Description
 
 ### HTTP request headers
 
-- **Content-Type**: application/x-www-form-urlencoded, 
-- **Accept**: application/json, 
+- **Content-Type**: application/x-www-form-urlencoded
```

Comment:
I thought these changes were part of a different PR?

Reply:
The twilio-go was not regenerated after those changes.

Reply:
@thinkingserious Can this PR not be merged? https://github.com/twilio/twilio-go/pull/67

Reply:
That PR was replaced with https://github.com/twilio/twilio-go/pull/72, once approved, will merge.